lib/OpenQA/Setup.pm

```diff
@@ -304,6 +306,13 @@ sub _validate_worker_timeout ($app) {
     }
 }
 
+sub _validate_security_policy ($app, $config) {
+    if ($config->{file_security_policy} !~ m/^(download-prompt|insecure-browsing)$/) {
```

Comment:
I would rather not include `insecure-browsing` at all. What's the use case for this?

Reply:
A setup only used by trusted users (no test scheduling based on PRs or the like) where the admin doesn't want to take the effort of setting up a subdomain. The subdomain feature is only present in the next PR anyway - so for this first step this is the *only* way to restore the current convenient behavior.

Besides, without this AC1 of the related ticket would not be fulfilled.

I also think the `insecure-...` makes it quite obvious that this is, well, insecure. So I wouldn't consider it problematic. (And also because it is/was the default for years now and good enough all that time.)

Reply:
The only thing I have to ask is whether would make sense to _highlight_ somehow that the app is "insecure" when the `insecure-browsing` is set. I am not asking for a change but maybe something to keep in mind for later

Reply:
> it is/was the default for years now

The ticket was filed because that's a terrible idea. Maybe we were lucky. Let's not use that as an argument 🙃

I don't get how you would argue convenience over security. But you have three people on your side, so that's completely fair.